NetKAN/Rescale-64.netkan

```diff
@@ -0,0 +1,25 @@
+{
+    "spec_version": "v1.4",
+    "install": [
+        {
+            "file": "GameData/RESCALE",
+            "install_to": "GameData"
+        }
+    ],
+    "depends": [
+        {
+            "name": "SigmaDimensions"
+        }
+    ],
+    "resources": {
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/163965-131-rescale-comprehensive-sd-configs-1027-10-oct-2017/"
+    },
+    "version": "1.0.2.7",
+    "ksp_version": "1.3.1",
+    "author": "Galileo88",
+    "download": "https://github.com/Galileo88/Rescale/releases/download/1.0.2.7/RESCALE_6.4.zip",
```

Comment:
`download` can be replaced by `"$kref": "#/ckan/github/Galileo88/Rescale/asset_match/RESCALE_6.4.zip"`